src/Cshrix/Data/Identifiers/Identifier.cs

```diff
@@ -360,7 +360,7 @@ public override bool Equals(object obj)
         /// <inheritdoc />
         /// <summary>Serves as the default hash function.</summary>
         /// <returns>A hash code for the current object.</returns>
-        public override int GetHashCode() => HashCode.Combine(Type, Sigil, Localpart, Domain);
+        public override int GetHashCode() => Utilities.HashCode.Combine(Type, Sigil, Localpart, Domain);
```

Comment:
With re-targeting to .NET 5, it should be possible to use the built-in [`HashCode.Combine`](https://docs.microsoft.com/en-us/dotnet/api/system.hashcode.combine?view=net-5.0) method

Reply:
OK - I thought that you were doing something special with GetHashCode here but if you're happy to use the built in, I'll remove your Utilities.HashCode.Combine completely.

Reply:
Aha, it's required for your custom Equals() method - do you want me to remove that too?

Reply:
Yeah I think that was added when the built-in HashCode was not available everywhere yet. All instances of the handrolled one should just be replaced with the one provided by dotnet.